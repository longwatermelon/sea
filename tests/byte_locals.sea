fn main() -> int {
    let x:byte = 10b;
    let y:byte = 20b;
    x = x + 5b;
    y = y - 3b;

    return x + y;
}
