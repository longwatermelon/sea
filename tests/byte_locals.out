RETURN_CODE:32
STDOUT:
